assets/stylesheets/bootstrap/mixins/_responsive-visibility.scss

```diff
@@ -8,6 +8,7 @@
     display: block !important;
```

Comment:
At line 20   , You can prove other action command :    `display: hidden !important; `